helm_deploy/laa-court-data-adaptor/values-dev.yaml

```diff
@@ -14,6 +14,7 @@ nameOverride: ""
 fullnameOverride: ""
 
 common_platform_url: https://hmcts-common-platform-mock-api.apps.live-1.cloud-platform.service.justice.gov.uk
+sqs_region: eu-west-2
```

Comment:
Is it worth adding this to the `values-test`, `values-uat`, `values-stage` and `values-prod` files now too, just so they don't get forgotten?

Reply:
Added